edb/schema/schema.py

```diff
@@ -82,6 +82,8 @@
     sn.UnqualName('std::pg'),
     sn.UnqualName('std::_test'),
     sn.UnqualName('std::fts'),
+    sn.UnqualName('std::lang'),
+    sn.UnqualName('std::lang::py'),
```

Comment:
Can you add `::js` / `::go` / `::rust` too?

But now I'm wondering... for Python we'll have just a fully qualified type name and then we can auto-import. For Go/JS/Rust the situation might be different...

@elprans @msullivan thoughts?

Reply:
Do we know what we want to put in js/go/rust?

Reply:
yeah sure